pulp_ansible/app/settings.py

```diff
@@ -24,3 +24,5 @@
 ANSIBLE_CONTENT_HOSTNAME = settings.CONTENT_ORIGIN + "/pulp/content"
 ANSIBLE_SIGNING_TASK_LIMITER = 10
 ANSIBLE_CERTS_DIR = "/etc/pulp/certs/"
```

Comment:
@gerrod3 would you be willing to open a PR to update the docs for these also maybe?